EmailNotifications/Email.cs

```diff
@@ -96,16 +96,16 @@ public string ConstructEmail()
             if (values.Length > 0)
             {
                 string[] styleProperties = {
-                "color:red;padding-top: 40px;",
-                "color:red;",
+                GetSeverityColor(values[2])+";padding-top: 40px;",
```

Comment:
This is a very confusing manner of getting values and will just be confusing even for yourself to maintain. 
Please have a look at implementing a key value pair object like a dictionary. Here is a link to [help out](https://www.dotnetperls.com/dictionary).